src/applications/vaos/tests/mocks/setup.js

```diff
@@ -42,6 +35,27 @@ export function createTestStore(initialState) {
   );
 }
 
+export function renderFromRoutes({ initialState, store = null, path = '' }) {
```

Comment:
So does this render the entire app now, instead of just a redux wrapped component? Looks good but would be interested to learn more about this approach during our next FE call.

Reply:
Yeah, it does, because the fetch is in the AppointmentsPage component. And to render AppointmentsPage with the tabs, you have to put it in a Route, which means that any children have to be sub-routes, and I though this was easier than writing fake partial routes for every test. At least this way we're testing our real code, even though I'd rather leave testing routing stuff to Cypress.